bylaws.tex

```diff
@@ -307,7 +307,7 @@
 If the member is still unsatisfied after being heard by the Executive Board, the appeal may be brought to the attention of the ResLife Advisor.
 
 \bsection{Expectations of House Members}
-Each member is required to pay House dues as stated in \ref{Collection of House Dues}, attend all House Meetings, and attend at least twenty-five (25) of the permanent director meetings.
+Each member is required to pay House dues as stated in \ref{Collection of House Dues}, attend all House Meetings, and attend at least twenty-five (25) of the permanent director meetings (including permanent and Ad-Hoc directorship meetings).
```

Comment:
Not quite.
https://github.com/ComputerScienceHouse/Constitution/pull/122/files#diff-315af85ab42c13dd83ff3940f2b3e9d7R310